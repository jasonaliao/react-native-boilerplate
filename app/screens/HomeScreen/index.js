
import React, { Component } from 'react';
import { Text, View, StatusBar } from 'react-native';

export default class HomeScreen extends React.Component {
    render() {
        return (
            <View style={{ flex: 1, justifyContent: 'center', alignItems: 'center' }}>
                <Text>Build somthing awesome</Text>
            </View>
        );
    }
}