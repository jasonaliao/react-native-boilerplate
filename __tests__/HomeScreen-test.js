import 'react-native'
import React from 'react'
import HomeScreen from '../app/screens/HomeScreen';

import renderer from 'react-test-renderer';

describe('Screens/HomeScreen', () => {
    it('it renders <HomeScreen /> correctly', () => {
        const tree = renderer.create(<HomeScreen />).toJSON();
        expect(tree).toMatchSnapshot();
    })
});