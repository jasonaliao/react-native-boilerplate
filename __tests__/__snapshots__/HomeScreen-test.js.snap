// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Screens/HomeScreen it renders <HomeScreen /> correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
  >
    Build somthing awesome
  </Text>
</View>
`;
